### DISCUSSION TITLE
How to bind ctrl+\ to split right?

### DISCUSSION THREAD

**Discussion by dangh** [raphamorim/rio#787] [Q&A] [answered]
This is my config. `Ctrl+-` can split down but `Ctrl+\` doesn't work. Not sure if this key was occupied by the terminal internally.

```toml
[bindings]
keys = [
  { key = '\\', with = 'control', action = 'SplitRight' },
  { key = '-', with = 'control', action = 'SplitDown' },
]
```

**Comment by raphamorim** [comment, marked as answer]
Hey @dangh , Does `\` work?

**Reply by dangh** [reply]
Goddam I thought backslash is escaping the closing quote 🤦 It's working now. Thank you @raphamorim

**Reply by raphamorim** [reply]
🙏

**Reply by raphamorim** [reply]
glad it works
